resources/helm/overlays/mesh-config/templates/enable-mesh-mtls.yaml

```diff
@@ -27,7 +27,7 @@ metadata:
     app: istio
     release: {{ .Release.Name }}
 spec:
-  host: "*.local"
+  host: "*{{ .Values.global.proxy.clusterDomain }}"
```

Comment:
```suggestion
  host: "*.{{ .Values.global.proxy.clusterDomain }}"
```

Reply:
Thank you! That must've slipped past me in this version